src/main/java/org/myrobotlab/jme3/Jme3Servo.java

```diff
@@ -19,19 +15,32 @@
  */
 public class Jme3Servo extends Jme3Object implements VirtualServo {
 
+  public Jme3Servo(Node node) {
+    super(node);
+    // TODO Auto-generated constructor stub
+  }
+
   // jme3 side
   transient Spatial node;
+  
+  transient Servo servo;
 
+  // FIXME - probably not needed
   int posUs;
 
+  // FIXME - probably not needed
   private int lastPosUs;
-
+  
+  
   // uber parent of all - moved up
   // transient SimpleApplication app;
+  // FIXME - should allow input of Servo & 3D Info + ref to Jme3App
+  public void someWeirdConstructor(Servo servo, Jme3App app) {

```

Comment:
shouldn't this be a ServoControl  not a Servo ?

Reply:
good catch .. fixed